### PR TITLE
Fix Mariabdb container not passing the healthcheck stage in docker-compose example

### LINKED ISSUE #245: Healthcheck fails for MariaDB container
When attempting to use a docker-compose file derived from the `docker-compose-mysql.yml` file, the MariaDB is indefinitely stuck on the healthcheck (and eventually stopped)

https://github.com/Clinical-Genomics/chanjo2/blob/54ce6104a6323df9702ae837860fc340d285ee66/docker-compose-mysql.yml#L15

Checking inside it, it seems `mysql` isn't present, which is used for the health check.

Updating it to `mariadb` I got it up and running:

`test: "mariadb --user=${MYSQL_USER} --password=${MYSQL_PASSWORD} --execute \"SHOW DATABASES;\"" `

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- Fix #245

### How to test:
- [x] Start a the app in a container connected to a dockerized mariadb following the instructions present in the [docs](https://github.com/Clinical-Genomics/chanjo2/blob/main/docs/deployment/docker-deployment.md#launching-the-app-connected-to-a-mysql-database-via-docker-compose)

### Expected outcome:
- [x] The app should launch

### Review:
- [x] Code approved by Jakob37
- [x] Tests executed by CR, Jakob37

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
Tested locally after modifying the template.env file like this:

```
MYSQL_USER=dbUser
MYSQL_PASSWORD=dbPassword
MYSQL_DATABASE_NAME=chanjo2_test
CONTAINER_DATA_VOLUME=/home/worker/DBs
HOST_DATA_VOLUME=./chanjodb
MYSQL_PORT=
MYSQL_HOST_NAME=mariadb
MYSQL_HOST_PORT=3307
CHANJO_HOST_PORT=8000
```

Main branch:

image


This branch:

image

File: docker-compose-mysql.yml [hunk 1]
```diff
@@ -10,9 +10,10 @@ services:
       - MYSQL_USER=${MYSQL_USER}
       - MYSQL_PASSWORD=${MYSQL_PASSWORD}
       - MYSQL_DATABASE=${MYSQL_DATABASE_NAME}
+      - MYSQL_HOST_PORT=3307
```
Comment (Jakob37) [inline docker-compose-mysql.yml:13]: Maybe `${MYSQL_HOST_PORT}` if connecting through the host?

(I connected through the subnet directly to the container, i.e. on port 3306)

**Review by Jakob37 — APPROVED** [review]
Looks good to me!

I have tested using the `mariadb` command, and it works on my machine.

File: docker-compose-mysql.yml (hunk 1 above)
Comment (northwestwitch) [reply docker-compose-mysql.yml:13]: Right, in line with the other ones! 👍🏻

**Pull request merged**